### ISSUE TITLE
Search / Description Enhancements

### ISSUE THREAD

**Issue by ryantaplin** [lmachens/arkesia.gg-web#107]
If I want to search for 'Hairplant', an item sold by a merchant, should I be able to find it if it's in a node description? 

It may be nice to be able to search for key words or items in descriptions (in the example above). To avoid complexity and bloating in the search maybe the easier solution is to implement a dictionary of important items/icons as suggested in misc and make them searchable?

As an example, maybe you implement something similar to what is displayed in the below image, where the reward icon is displayed and styled in a unique way, and is also searchable.
![image](https://user-images.githubusercontent.com/11903234/159361835-0d269fc0-20dc-43ec-9ecf-33ca28c5f42b.png)

**Comment by lmachens** [comment]
Sounds good!
The difficult part is to add searchable icons.
This sounds like a functionality of a Lost Ark database. Maybe we can link items of an existing database?

**Comment by ryantaplin** [comment]
I agree, although I would like to build other tools for lost ark (like tracking collectibles, quests etc) it is a lot of additional work.

I was a bit underwhelmed by lostarkhub and hoped they had a database section... but doesn't mean we can't try partner/work with another website that has databases.
